faq.html

```diff
@@ -213,16 +213,10 @@
 					</div>
 				</div>
 				
-				<div class="faq" onclick="toggle('faq26')">
-					<strong>Q:</strong> What is IntelDeflater and how do I control it?
```

Comment:
I think we should keep the documentation line on this but modify the text to reflect the fact that ID has been taken out into a separate repo (https://github.com/Intel-HLS/GKL), and specify when this happened, so people using older version don't get too confused.